lib/generators/templates/setup/brewfile

```diff
@@ -0,0 +1,9 @@
+# https://formulae.brew.sh/formula/vips
+brew "vips"
+
+# https://formulae.brew.sh/formula/redis
+brew "redis"
```

Comment:
I wonder if these should be conditionally loaded based on the Gemfile?

Reply:
After giving it some thought, I think it's acceptable to leave these all in. We're focusing on new applications, and most new applications will need image processing and Redis.